view/frontend/templates/js/replacejs.phtml

```diff
@@ -1714,16 +1714,50 @@ if (!$block->isSaveCartInSections()) { ?>
                 if (waitingForResolvingPromises && (localCallNumber === countConfigureCalls)) {
                     customerData.reload(['boltcart'], true);
                 }
-            }, 5000);
+            }, 10000);
         }
 
         var invalidateBoltCart = function() {
             customerData.reload(['boltcart'], true);
-            callConfigureWithPromises();
+            if (!waitingForResolvingPromises) {
+                callConfigureWithPromises();
+            }
+        }
+
+        var magentoCart;
+        var boltCartDataID;
+
+        var invalidateBoltCartIfOutdated = function() {
+            setTimeout(function () {
```

Comment:
We need to run this code after all code from the JS stack executed. Is timeout 1 is the proper way to do it? @daisy1754

Reply:
what do you mean by "all code from the JS stack"?

Reply:
I needed to say **event loop**. We need to add code in the end of JS event loop